### PR TITLE
add registration test WINNF.FT.S.REG.16

### PR DESCRIPTION
**Pull request by janetliuboston**

### CODE REVIEW

File: src/harness/testcases/registration_testcase.py [hunk 1]
```diff
@@ -543,6 +543,80 @@ def test_WINFF_FT_S_REG_13(self):
...
+    conditionals_a = {'registrationData': [
+        {'cbsdCategory': device_a['cbsdCategory'],
+         'fccId': device_a['fccId'],
+         'cbsdSerialNumber': device_a['cbsdSerialNumber'],
+         'airInterface': device_a['airInterface'],
+         'installationParam': device_a['installationParam']}
+    ]}
+
+    # Device 2 Cat A out-of-range or the wrong type azimuth
+    self.assertEqual(device_a['cbsdCategory'], 'A')
```
Comment (kate-harrison) [inline src/harness/testcases/registration_testcase.py]: I think you mean device_c here.

File: src/harness/testcases/registration_testcase.py [hunk 2]
```diff
@@ -543,6 +543,80 @@ def test_WINFF_FT_S_REG_13(self):
...
+        {'cbsdCategory': device_c['cbsdCategory'],
+         'fccId': device_c['fccId'],
+         'cbsdSerialNumber': device_c['cbsdSerialNumber'],
+         'airInterface': device_c['airInterface'],
+         'installationParam': device_c['installationParam']}
+    ]}
+    conditionals_c['registrationData'][0]['installationParam']['azimuth'] = -1
+
+    # Device 3 Cat A out-of-range, or the wrong Type value for latitude.
+    self.assertEqual(device_a['cbsdCategory'], 'A')
```
Comment (kate-harrison) [inline src/harness/testcases/registration_testcase.py]: device_e?

File: src/harness/testcases/registration_testcase.py [hunk 3]
```diff
@@ -543,6 +543,80 @@ def test_WINFF_FT_S_REG_13(self):
...
+    conditionals_e = {'registrationData': [
+        {'cbsdCategory': device_e['cbsdCategory'],
+         'fccId': device_e['fccId'],
+         'cbsdSerialNumber': device_e['cbsdSerialNumber'],
+         'airInterface': device_e['airInterface'],
+         'installationParam': device_e['installationParam']}
+    ]}
+    conditionals_e['registrationData'][0]['installationParam']['latitude'] = 'a'
```
Comment (kate-harrison) [inline src/harness/testcases/registration_testcase.py]: Maybe use 91 here, same as in REG.15?

File: src/harness/testcases/registration_testcase.py (hunk 2 above)
Comment (janetliuboston) [inline src/harness/testcases/registration_testcase.py]: Yes, this should be device_e.

File: src/harness/testcases/registration_testcase.py (hunk 1 above)
Comment (janetliuboston) [inline src/harness/testcases/registration_testcase.py]: Yes, this should be device_c.

File: src/harness/testcases/registration_testcase.py (hunk 3 above)
Comment (janetliuboston) [inline src/harness/testcases/registration_testcase.py]: Yes.

**Pull request merged**